Formula/x/xk6.rb

```diff
@@ -21,6 +21,8 @@ class Xk6 < Formula
   end
 
   depends_on "go"
+  depends_on "gosec"
+  depends_on "govulncheck"
```

Comment:
What are these needed for? And can you add a test to make sure they work?

Reply:
Thanks for the comment. I've written a test for it now.